ocs_ci/ocs/disaster_recovery/acm.py

```diff
@@ -0,0 +1,160 @@
+import logging
+import time
+
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.support import expected_conditions as ec
+
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.ui.acm_ui import AcmPageNavigator
+from ocs_ci.ocs.ui.views import locators
+from ocs_ci.ocs.ui.base_ui import login_ui
+
+
+log = logging.getLogger(__name__)
+
+# Lines below will be moved to config files
+CLUSTER_NAME_1 = "Cluster-a"
+CLUSTER_NAME_2 = "Cluster-b"
+KUBECONFIG_A = "TBD"
+KUBECONFIG_B = "TBD"
+###########################################
+
+
+#####################################################
+# The Code below intended to run on ACM Hub cluster #
+#####################################################
+
+
+class AcmAddClusters(AcmPageNavigator):
+    """
+    ACM Page Navigator Class
+
+    """
+
+    def __init__(self, driver):
+        super().__init__(driver)
+        self.page_nav = locators[self.ocp_version]["acm_page"]
+
+    def import_cluster(self, cluster_name, kubeconfig_location):
+        """
+
+        Args:
+            cluster_name (str): cluster name to import
+            kubeconfig_location (str): kubeconfig file location of imported cluster
+
+        """
+        self.navigate_clusters_page()
+        time.sleep(3)
```

Comment:
changes look ok , we should see how we can eliminate sleeps in next PR

Reply:
@vasukulkarni We have multiple functions in base_ui.py to avoid hardcoded sleep.